Update tutorial-configure-est-server.md (#112335)
Azure/azure-content: articles/iot-edge/tutorial-configure-est-server.md

1. In the [Azure portal](https://portal.azure.com), navigate to your instance of IoT Hub Device Provisioning Service.
1. Under **Settings**, select **Manage enrollments**.
1. Select **Add enrollment group** then complete the following steps to configure the enrollment.
1. On the **Registration + provisioning** tab, choose the following settings:

>     :::image type="content" source="./media/tutorial-configure-est-server/dps-add-enrollment-latest.png" alt-text="A screenshot adding DPS enrollment group using the Azure portal.":::

Reviewer: ```suggestion
    :::image type="content" source="./media/tutorial-configure-est-server/device-provisioning-service-add-enrollment-latest.png" alt-text="A screenshot adding DPS enrollment group using the Azure portal.":::
```

1. Under **Settings**, select **Manage enrollments**.

> 1. Select **Add enrollment group** then complete the following steps to configure the enrollment:

Reviewer 2: ```suggestion
1. Select **Add enrollment group** then complete the following steps to configure the enrollment.
1. On the **Registration + provisioning** tab, choose the following settings:
```

    :::image type="content" source="./media/tutorial-configure-est-server/dps-add-enrollment-latest.png" alt-text="A screenshot adding DPS enrollment group using the Azure portal.":::

    |Setting | Value |
    |--------|---------|

>     |Attestation Type | Select **X.509 certificates uploaded to this Device Provisioning Service Instance** |

Reviewer 2: ```suggestion
    |Attestation mechanism| Select **X.509 certificates uploaded to this Device Provisioning Service instance** |
```

    |Primary certificate | Choose your certificate from the dropdown list |
    |Group name | Provide a friendly name for this group enrollment |

>     |Enable this enrollment | Select the checkbox |

Reviewer 2: ```suggestion
    |Provisioning status | Select **Enable this enrollment** checkbox |
```

>     |Target IoT Hubs | From IoT Hubs tab, choose your IoT Hub from the dropdown list |
>     |Enable IoT Edge on provisioned devices | From Device settings tab, select the checkbox |

Reviewer 2: ```suggestion

1. On the **IoT hubs** tab, choose your IoT Hub from the list.
1. On the **Device settings** tab, select the **Enable IoT Edge on provisioned devices** checkbox.
```

    
    The other settings aren't relevant to the tutorial. You can accept the default settings.

> 

Reviewer 2: I can't add a suggestion to the next line.  Instead of Select **Save**, change to Select **Review + create**.
